### PR TITLE
bluetooth: rpc: Add host missing callback handlers

### PR DESCRIPTION
**Pull request by KAGA164**
This adds missing callback handlers for a CCC descriptor write and match operation on the host side.

### CODE REVIEW

**Review by grochu — APPROVED** [review]
The changes look good.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -182,6 +182,12 @@ Bluetooth libraries and services
 
   * Fixed discovery of empty services.
 
+* :ref:`ble_rpc` library:
+
+  * Added missing host callback handlers for the CCC descriptor write and match operation.
+  * Fixed the attribute write callback serialization.
+  * Fixed the GATT dynamic service unregister function call serialization.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:189]: These are all a bit hard to read: Which is an attribute to what. Having long list of substantives makes it difficult to understand.
Could these be for example something like this?

* Added (do we need to say missing, if we added something) host callback handlers for the ``write`` and ``match`` operations of the CCC descriptor.
* Fixed the serialization of the ``write`` callback attribute.
* Fixed the serialization of ... (cannot really say what defines what)

I may have misunderstood all of this totally.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: I rewrited it a little. I hope it is understandable now

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -199,6 +199,12 @@ Bluetooth libraries and services
 
   * Fixed discovery of empty services.
 
+* :ref:`ble_rpc` library:
+
+  * Added host callback handlers for the ``write`` and ``match`` operations of the CCC descriptor.
+  * Fixed the serialization of the GATT attribute ``write`` callback.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:205]: A lot better now. But still a bit unsure about this one. What is serialized? 
- Is it the ``write`` callback applied to the GATT attribute?
- Or something totally different?

Sorry for not understanding the coding world.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (peknis) [reply doc/nrf/releases/release-notes-changelog.rst:205]: Based on discussion in Teams, maybe something like this then:

Fixed the serialization of the ``write`` callback applied to the GATT attribute.

Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: Ok, done

**Review by peknis — APPROVED** [review]

**Comment by DatGizmo** [comment]
With the new Integration Test Specification process we don't need the CI-* github labels.
I will remove them.

To get the test spec feature working correctly on this PR, you need to rebase it to the latest main to get the [.github/test-spec.yml](https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml) file

Then run CI here to update the test spec.

[Information about the test spec feature](https://projecttools.nordicsemi.no/confluence/display/NCS/Test+Spec)
[CI-Help for issues](https://ci-help.nordicsemi.no/)

**Pull request merged**